algorithms/landmark-triangulation/include/landmark-triangulation/landmark-triangulation.h

```diff
@@ -20,6 +20,9 @@ void retriangulateLandmarksAlongMissionAfterVertex(
     const pose_graph::VertexId& starting_vertex_id, vi_map::VIMap* map);
 void retriangulateLandmarksOfVertex(
     const pose_graph::VertexId& storing_vertex_id, vi_map::VIMap* map);
+void retriangulateLandmarks(
```

Comment:
The whole code bit in landmark triangulation I think we can avoid duplicating, no? I guess why you did it this way is because you only have a `retriangulateLandmarksOfVertex` function and not one for only a specific list of landmark ids. But better would be to make indeed the retriangulateLandmarks you have and then simply have the previous function for triangulating all landmarks from a vertex call the new one with the list of all landmarks in that vertex

Reply:
you're absolutely right and this is what I actually planned to do but forgot to replace it in the end, fixed now